.gitignore

```diff
@@ -12,9 +12,12 @@ typo3_src
 web/index.php
 web/typo3
 
-web/typo3conf/ext
+web/typo3conf/
+!web/typo3conf/ext
 
 web/fileadmin/_temp_
 web/fileadmin/user_upload/_temp_
 
 typo3temp
+
+extensions
```

Comment:
Hm, this fights the `.gitkeep` that I added to `extensions/` just to keep the directory structure. But yes, then the DDB extension annoyingly pops up in `git status`.

So I'd suggest to remove `extensions/.gitkeep` and add `mkdir -p extensions/` to the quickstart script.

Reply:
I think we can keep this behaviour. The `.gitkeep` makes sure, the directory exist on checkout. The `.gitignore` makes sure, nothing will be committed. 
What do you think?

Reply:
I'm not sure. What concerns me is that then the state of the working tree would depend on the fact that this has happened in two commits. (If someone were to take and re-initialize the repo, the `.gitkeep` wouldn't be committed anymore, and such a difference shouldn't occur IMHO.)

Reply:
Alternatively, perhaps put a `.gitignore` file into `extensions/` that says

```
*
!.gitignore
```

?

Reply:
> Alternatively, perhaps put a `.gitignore` file into `extensions/` that says
> 
> ```
> *
> !.gitignore
> ```
> 
> ?

Maybe this is the solution. Not sure about the perfect syntax. Seems again to work for me.